### DISCUSSION TITLE
Different results between autogluon predictor and sklearn predictor

### DISCUSSION THREAD

**Discussion by gkositsk** [autogluon/autogluon#2768] [Q&A] [answered]
After training autogluon model on data, I extract the original sklearn random forest model.
When calculating the score for validation data with the sklearn model, I got a different results than the score i got in the leaderboard.
anyone has idea why it happens? Note that I set feature_generator=None, i didn't want any additional feature engineering.

**Comment by Innixma** [comment, marked as answer]
Refer to the FAQ: [How can I reproduce the results of model XYZ without using AutoGluon?](https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-faq.html#how-can-i-reproduce-the-results-of-model-xyz-without-using-autogluon)

**Comment by Nahian099** [comment]
The link in the answer is unavailable due to access denied

**Reply by eric15342335** [reply]
https://auto.gluon.ai/stable/tutorials/tabular/tabular-faq.html#how-can-i-reproduce-the-results-of-model-xyz-without-using-autogluon
